### ISSUE TITLE
A Bar file should be importable.

### ISSUE THREAD

**Issue by Dvs1010** [flowable/flowable-engine#552]
We have two options to export application project, 1) As a ZIP 2) As a BAR. But we can import only ZIP file since it accepts only zip while importing. So exported BAR file becomes useless if we cant import it. So, there should be a functionality to import a bar file too.

![issue](https://user-images.githubusercontent.com/6284881/30047374-73c1bbb0-922e-11e7-91ea-26b0b44d8a47.png)

**Comment by tijsrademakers** [comment]
Yes, the download as BAR is to deploy the application on a Flowable Engine. The ZIP file is to transfer the app to another modeling environment. But I agree that it should be possible to import a BAR file as well.
